src/pseudopeople/interface.py

```diff
@@ -228,12 +226,17 @@ def generate_decennial_census(
 
     :raises ConfigurationError:
 
-        An incorrect config is provided.
+        An invalid config is provided.
 
     :raises DataSourceError:
 
-        An incorrect pseudopeople simulated population data source is
+        An invalid pseudopeople simulated population data source is
         provided.
+
+    :raises ValueError:
+
+        The simulated population has no data for the specified year or
+        state.
```

Comment:
Hm... this feels a little unclear because it doesn't sound like it would be dataset-specific. How would you feel about something like this?

```suggestion
        The simulated population has no data for this dataset in the specified year or
        state.
```